packages/components/src/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from "./engine"
-export { schema } from "./interfaces"
+// export { schema } from "./interfaces"
```

Comment:
Hmm the schema is part of the components package but it is meant to be the public interface for actually using the components. Exporting here so that we can do a single Ajv.validate instance on the entire page schema using this Isomer JSON Schema.